### PR TITLE
Adding Dominic's Info to people.yml 

### PR DESCRIPTION
**Pull request by dpeters001**
Adding Dominic's information to the people.yml file in public repository

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -858,6 +858,33 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: scientist
+dpeters:
+  active: true
+  aka: Dom
+  avatar: https://github.com/dpeters001.png
+  bio: Dominic is a rising-senior and chemical engineering undergraduate with minors in
+  mathematics and business administration. He has experience with Linux-based and
```
Comment (sbillinge) [inline db/people.yml:866]: indent this block by a further two whitespaces (indentation has meaning in yaml)

File: db/people.yml [hunk 2]
```diff
@@ -858,6 +858,33 @@ cyshi:
...
+  python coding languages/protocols. His previous research experience includes protein
+  engineering of the SARS-CoV-2 spike protein and the mapping of energetic landscapes of 
+  viral pathogens. Dominic enjoys electronic music production, 10K races, and
+  hiking in his free time. 
+  education:
+    - begin_year: 2019
+      degree: Chemical Engineering Undergraduate
```
Comment (sbillinge) [inline db/people.yml:873]: Probably "BSc in Chemical Engineering"?  something like that?

File: db/people.yml [hunk 3]
```diff
@@ -858,6 +858,33 @@ cyshi:
...
+      institution: The City College of New York
+      end_year: 2023
+  employment:
+    - begin_year: 2020
```
Comment (sbillinge) [inline db/people.yml:877]: This seems to be a duplicate key (see below) and shoudl be 2022

File: db/people.yml [hunk 4]
```diff
@@ -858,6 +858,33 @@ cyshi:
...
+      organization: Center for Discovery and Innovation
+      position: Undergraduate Researcher
+      begin_year: 2022
+      begin_month: 6
+      position: Amazon SURE Research Fellow
```
Comment (sbillinge) [inline db/people.yml:882]: this is also a duplicate key.  Pick which description you like better and keep it.

File: db/people.yml [hunk 5]
```diff
@@ -858,6 +858,33 @@ cyshi:
...
+      organization: Columbia U
```
Comment (sbillinge) [inline db/people.yml:883]: this should be a tag that is dereferenced in another institutions.yml collection, so set to `columbiau`

File: db/people.yml [hunk 6]
```diff
@@ -858,6 +858,33 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: scientist
+dpeters:
+  active: true
+  aka: Dom
```
Comment (sbillinge) [inline db/people.yml:863]: this should be a list, not a string, so it would look like
```
aka:
  - Dom
 ```

**Review by sbillinge — COMMENTED** [review]
Great job getting this PR in.....lots to learn there probably to get it done, so I am super happy to see it.   Please see the inline comments addressing some issues.  There may be more that I didn't see so take a new look over everything

**Comment by sbillinge** [comment]
These edits look good but this file is now conflicted with other changes to files that have been merged by others (don't worry, it happens).

One comment....please go through and say "done" or respond in some way to each inline comment.  This is a general comment, but it makes the review easier because the reviewer then knows you have seen the comment and what you think about it.

But to resolve the conflict, check out and synchronize your local master with upstream, then re-check out his branch, then merge master into this branch.  It will raise merge conflicts that you have to resolve by hand (be careful deleting things that you didn't edit).  Then commit and push again.

File: db/people.yml (hunk 2 above)
Comment (dpeters001) [reply db/people.yml]: Changed to "BE Chemical Engineering"

File: db/people.yml (hunk 3 above)
Comment (dpeters001) [reply db/people.yml]: Removed.

File: db/people.yml (hunk 4 above)
Comment (dpeters001) [reply db/people.yml:880]: Done.

File: db/people.yml (hunk 5 above)
Comment (dpeters001) [reply db/people.yml]: Done.

File: db/people.yml (hunk 6 above)
Comment (dpeters001) [reply db/people.yml]: Done.

**Comment by sbillinge** [comment]
thanks @dpeters001

**Pull request closed without merging**